integration/tests/cook/mesos.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import sys
+from urllib.parse import urlparse, parse_qs
+
+
+def instance_to_agent_url(instance):
+    """Given a job instance, returns the base Mesos agent URL, e.g. http://agent123.example.com:5051"""
+    hostname = instance['hostname']
```

Comment:
The `hostname` can be moved to the else branch.

Reply:
Done